zulipterminal/cli/run.py

```diff
@@ -322,11 +371,48 @@ def _write_zuliprc(
         return f"{ex.__class__.__name__}: zuliprc could not be created at {to_path}"
 
 
-def parse_zuliprc(zuliprc_str: str) -> Dict[str, SettingData]:
-    zuliprc_path = path.expanduser(zuliprc_str)
-    while not path.exists(zuliprc_path):
+def check_for_default_zuliprc() -> str:
+    zuliprc_locations = [
+        DOWNLOADED_PATH_ZULIPRC,
+        HOME_PATH_ZULIPRC,
+    ]
+    valid_locations = [
+        location for location in zuliprc_locations if Path(location).exists()
+    ]
+    valid_locations.extend(
+        str(file) for file in Path(ZULIP_CONFIG_PATH).glob("zuliprc")
+    )
+    valid_locations.extend(
+        str(file) for file in Path(ZULIP_CONFIG_PATH).glob("*/zuliprc")
+    )
+
+    count = len(valid_locations)
+    if count > 1:
+        valid_locations_string = "\n".join(valid_locations)
+        exit_with_error(
+            f"Found multiple zuliprc files at:\n{valid_locations_string}\n"
+            "Please retry by specifying the path to your target zuliprc file."
+        )
+    elif count == 1:
+        return valid_locations[0]
+    return ""
```

Comment:
This implementation seems great! However, should we consider giving priority to `HOME_PATH_ZULIPRC` if multiple configuration files are found, since that was previously the location of zuliprc?

If users want to use a different configuration file instead of the default, they can specify a specific target zuliprc file to override the default. We can notify the config file used while loading ZT, similar to how we do with the settings.

Alternatively, instead of exiting with an error, we could include an index and prompt the user to choose which organization they would like to log into.

For example:

```shell
Choose which organization you'd like to log in to (Enter the number):
1: Zulip Community: chat.zulip.org
2: CTF Chat: ctf.zulipchat.com
3: Example: example.zulip.com
```